CONTRIBUTING.md

```diff
@@ -35,22 +35,54 @@ Contributions are essential for keeping this extension great. We try to keep it
 	```bash
 	$ mvnw.cmd verify
 	```
-       
+
 
 5. `cd vscode-xml/`
-   
-6. Install the dependencies:  
-	```bash  
+
+6. Install the dependencies:
+	```bash
 	$ npm install
 	```
 
 7. In `vscode-xml/`, build the server by running:
 
-	```bash   
+	```bash
 	$ npm run build-server
 	```
 
 8. To run the extension, open the Debugging tab in VSCode.
 9. Select and run 'Launch Extension (vscode-xml)' at the top left:
 
     ![ Launch Extension ](./images/LaunchExtension.png)
+
+## Binary server development
+
+### Testing a binary version of LemMinX
+
+1. Copy the binary version of LemMinX to:
+
+   | OS | Location (relative to root of repository) |
+   | --- | --- |
+   | Linux | `./server/lemminx-linux` |
+   | macOS | `./server/lemminx-darwin-x86_64` |
+   | Windows | `.\server\lemminx-win32.exe` |
+
+2. Make sure that you set `xml.server.preferBinary` to `true`,
+and disable any [LemMinX extensions](./docs/Extensions.md)
```

Comment:
I would remove this 'and', leaving the comma, since you use another one a little while later in this sentence below.